examples/muenster/analysis/test_hdf5.py

```diff
@@ -0,0 +1,6 @@
+__author__ = 'axel'
+import h5py
+import tables
+import pandas as pd
+filename    = '/home/axel/PycharmProjects/pax/tpc_kr_150410_8k.hdf5'
+df = pd.read_hdf(filename,"Hit")
```

Comment:
__[flake8]__
* [E231] missing whitespace after ','
* [W292] no newline at end of file